test/e2e/storage/volumes.go

```diff
@@ -503,24 +503,21 @@ var _ = utils.SIGDescribe("Volumes", func() {
 	Describe("vsphere [Feature:Volumes]", func() {
 		It("should be mountable", func() {
 			framework.SkipUnlessProviderIs("vsphere")
+			vspheretest.Bootstrap(f)
+			nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
+			Expect(nodeList.Items).NotTo(BeEmpty(), "Unable to find ready and schedulable Node")
+			nodeInfo := vspheretest.TestContext.NodeMapper.GetNodeInfo(nodeList.Items[0].Name)
+
```

Comment:
*If* the above block is common to the vsphere tests then consider moving it to a helper func and maybe into a `BeforeEach`.